packages/flutter/lib/src/material/expansion_tile.dart

```diff
@@ -85,9 +86,12 @@ class ExpansionTile extends StatefulWidget {
   /// The color to display behind the sublist when expanded.
   final Color? backgroundColor;
 
-  /// A widget to display instead of a rotating arrow icon.
+  /// A widget to display instead of an arrow icon.
   final Widget? trailing;
 
+  /// Specifies if the trailing is rotating (true, the default) or not (false).
+  final bool rotatingTrailing;
```

Comment:
I believe the original UX was as following:

1) If `trailing` was provided, then no rotation was applied.
2) If no `trailling` was provided, then a rotating arrow would appear.

I think the pr should keep the same behavior.

Reply:
ooh, good catch! I wonder why the existing tests did not spot it.

Reply:
Yeah, they probably didn't add a test for that.

Reply:
yup! and now I have added it! :)